mil_perception/include/mil_vision_lib/cv_tools.hpp

```diff
@@ -0,0 +1,26 @@
+#include <string>
+#include <vector>
+#include <opencv2/opencv.hpp>
```

Comment:
These kind of libraries usually take the longest to compile since they use so many things that are pretty complex on their own. I'd recommend not using this convenience include but instead including the things that are actually used.